### DISCUSSION TITLE
Add API Field for Repository Public Date

### DISCUSSION THREAD

**Discussion by ramoures** [community/community#140713] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hello,

Currently, when we retrieve repository information using the API, we can obtain the created_at date, which indicates when the repository was created. However, this date does not reflect when the repository was made public, as it might have been private for several days before being publicly accessible.

As the API currently lacks a specific field that indicates the date when a repository was changed from private to public, it would be immensely helpful if you could consider adding this functionality.

Thank you!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ramoures , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!
